scripts/network-launch/launch.sh

```diff
@@ -86,8 +90,11 @@ start_node () {
       bootNodeID=$(curl -H 'Content-Type: application/json' --data '{ "jsonrpc":"2.0", "method":"system_localPeerId", "id":1 }' https://${bootNodeHost}:9934 -s | jq '.result')
       sleep 5
   done
+
+  clone_scripts ${ip}
+  stop_node_partially ${ip}
```

Comment:
Why do we need stop here?

Reply:
I just want to make sure the node is not running.
Let's say you downloaded the necessary backup to the root of your repo.
You can simply run the command with the "backup" flag and it will ensure that:
- repo is not cloned again
- node is not operating